src/pybind/mgr/dashboard/services/feedback.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+from typing import Optional
+from enum import Enum
+import errno
+import json
+import requests
+
+from requests.auth import AuthBase
+from mgr_module import CLICommand, CLIWriteCommand, CLIReadCommand, HandleCommandResult
+
+from .. import mgr
+from ..settings import Settings
+from ..exceptions import DashboardException, ScopeNotInRole
+from ..rest_client import RestClient
+from ..model.feedback import Feedback
+
+class config:
+    url = 'tracker.ceph.com'
+    port = 443
+
+
+class RedmineAuth(AuthBase):
+    def __init__(self, access_key):
+        self.access_key = str(access_key)
+
+    def __call__(self, r):
+        r.headers['X-Redmine-API-Key'] = self.access_key
+        return r
+
+
+class CephTrackerClient(RestClient):
+    access_key = ''
+
+    def __init__(self):
+        super().__init__(config.url, config.port, client_name='CephTracker',
+                         ssl=True, auth=RedmineAuth(self.get_api_key()), ssl_verify=True)
+
+    @staticmethod
+    def get_api_key():
+        try:
+            access_key = Settings.CEPH_TRACKER_API_KEY
+        except KeyError as error:
+            raise DashboardException(msg='',
```

Comment:
Please set a more useful error message here, so that the user knows that the `CEPH_TRACKER_API_KEY` setting is not set.